### ISSUE TITLE
Add option to render link to file as well

### ISSUE THREAD

**Issue by netzdoktor** [dylanowen/mdbook-graphviz#16]
If `output-to-file = true`, it would be great to also have the option to render a link to the file as well (so a user can click on it to see the larger version).

So instead of

```md
![](f.generated.svg)
```

One would generate:

```md
[![](f.generated.svg)](f.generated.svg)
```

What would be a way forward to add this? I guess changing the output should be straightforward, but what would be a good way to enable/disable this behaviour? In my opinion, it should only be available if `output-to-file` is set to true and probably be called `link-to-file`.

What do you think?

**Comment by Cypher1** [comment]
This should work if you download and install the branch associated with the PR I just put up.
I'm using it at
(Work in progress) 
blog.mimir.systems/posts/Lambda/2023-04-20-lambda-calc.html
if you'd like a preview or to make any suggestions.

**Comment by Cypher1** [comment]
@dylanowen Could I get a review for this? Thx

**Comment by Cypher1** [comment]
This has been merged and released under v0.2.0 !
